apps/src/templates/export/expo/CustomAsset.exported_js

```diff
@@ -158,7 +158,7 @@ export default class CustomAsset {
     }
     this.downloading = true;
     try {
-      const localUri = `${FS.cacheDirectory}${this.name}`;
+      const localUri = `${FS.cacheDirectory}${encodeURI(this.name)}`;
```

Comment:
Have you tested this, actually exporting an app with a space in an asset filename?  I wonder if doing this encode can break references to assets in student source code.

Reply:
Yes, I tested it. Before, we couldn’t place the file in local storage, so the web view couldn’t find it. Now, we can and it works.